### ISSUE TITLE
Rival 3 mouse

### ISSUE THREAD

**Issue by ghost** [bitraker/SteelSeriesScreenSync#5]
First of all, thank you for trying to make something like this. I have razer keyboard which only needs me to run Synapse and an exe to screen sync my keyboard. I want my mouse to sync too. I dont have any basic in programming and I dont know python but I installed it, using 3.11.5, and other modules I only use like "pip install Pillow" so the newest version installed. I ran the exe, it only consist half of the screenshot you show, until the "Dark Mode", without the "Start in Tray" and below. I do believe it is supposed to work on mouse too (like you mention your mouse). Is it because I dont use the exact version? or am I missing something? Thank you in advance.

also, I only unzip the first main which has exe (not the others with 0.2 to 0.4) and the entire "code" I downloaded.
here is the application log:
2023-09-01T20:21:54.517753 informational Starting application
2023-09-01T20:21:56.619703 error failed registering game
2023-09-01T20:21:56.620700 critical exception in Y:\SteelSeries - Screen Color Sync\build-files\.\steelseries.py at 90: : HTTPConnectionPool(host='127.0.0.1', port=51017): Max retries exceeded with url: /game_metadata (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000013A1A42DB70>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))

update: installed correct version of the module except the pywin32 bcs 302 cant be found. (using 306). Still no clue.

**Comment by bitraker** [comment]
Is your mouse a SteelSeries ?

The software in this repository depends on SteelSeries GG software to be installed in order to work.

**Comment by rimmetz17** [comment]
Yes it is steelseries and I do have a steelseries GG. I just realized I installed the modules, but is "run it" different from that? how do I run a module?.

**Comment by rimmetz17** [comment]
Yes it is steelseries and I do have a steelseries GG. I just realized I installed the modules, but is "run it" different from that? how do I run a module?.

**Comment by ghost** [comment]
Yes it is steelseries and I do have a steelseries GG. I just realized I installed the modules, but is "run it" different from that? how do I run a module?.

update:
I ran the module using "python -m Pillow" and it says "no module named Pillow"
tried to reinstall it, but it says "requirement already satisfied"

**Comment by bitraker** [comment]
Okay that sounds strange. Perhaps the pip you install with is for another version of python.

If you run python3, you should install with pip3 install <module>

You can also try and download the SteelSeries v04 zip from this repository, that's a previous version, but it is compiled with all modules and there is an exe file you can run.

**Comment by ghost** [comment]
I'll try to start over with pip3 thank you
